wellregistry/registry/admin.py

```diff
@@ -16,6 +16,18 @@ class RegistryAdminForm(forms.ModelForm):
     """
     Registry admin form.
     """
+    def clean(self):
+        """
+        Override form clean to do multi field validation
+        """
+        cleaned_data = super().clean()
+        if (cleaned_data['display_flag'] and cleaned_data['wl_sn_flag']) and \
```

Comment:
This will be the appropriate place to add multi-field validations.

Reply:
Great